pkg/api/Makefile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BIN=api
```

Comment:
I'm not familiar with these build scripts, but it appears that the Makefile you linked is behaving differently than this one (see below).

Without `BIN`:

```console
ENG000656:service-catalog aaronschlesinger$ make build
Building github.com/kubernetes-incubator/service-catalog/contrib/registry
Building github.com/kubernetes-incubator/service-catalog/contrib/broker/k8s
Building github.com/kubernetes-incubator/service-catalog/contrib/broker/user_provided
Building github.com/kubernetes-incubator/service-catalog/controller
Building github.com/kubernetes-incubator/service-catalog/pkg
```

With `BIN`:

```console
ENG000656:service-catalog aaronschlesinger$ make build
Building github.com/kubernetes-incubator/service-catalog/contrib/registry
Building github.com/kubernetes-incubator/service-catalog/contrib/broker/k8s
Building github.com/kubernetes-incubator/service-catalog/contrib/broker/user_provided
Building github.com/kubernetes-incubator/service-catalog/controller
Building github.com/kubernetes-incubator/service-catalog/pkg
Building github.com/kubernetes-incubator/service-catalog/pkg/api
```